src/repeats/Header/Nav.js

```diff
@@ -1,13 +1,31 @@
 import React from "react";
+import "./Header.css"
 import { Link } from "react-router-dom";
 
 const HeaderNav = () => {
   return (
-    <nav>
-      <Link to="/">Home</Link> <Link to="/about">About</Link>{" "}
-      <Link to="/contact">Contact</Link>{" "}
-      <Link to="/services">Towing Services</Link>
-    </nav>
+    <navcont>
+      <info className="info">
+        <div className="infoLeft">
+          <div className="sales">SALES, PARTS & SANDBLASTING</div>
+          <div className="phoneNumber">(908) 361-0376</div>
+        </div>
+        <img className="logo" src="" alt="United Truck Service Logo"></img>
+        <div className="infoRight">
+        <div className="towing">TOWING</div>
+        <div className="phoneNumber">(908) 224-0060</div>
+        </div>
+      </info>
```

Comment:
I believe lines 7-18 can be it's own component. A `Content.js` would suffice within `repeats/Header`. 
You would then reference `Content.js` within the header container.